localization/french/culture/wc_cultural_maa_innovations_l_french.yml

```
@@ -90,10 +90,10 @@
  innovation_ursa:0 "Ursas"
  innovation_ursa_desc:0 "Les Ursas sont les plus grands, les plus puissants et les plus féroces des furbolgs. Ils dominent les humains et les orcs et peuvent briser le cou des elfes d'un seul coup de patte. Ils sont immensément forts et résistants, et se concentrent sur l'amélioration de leur comportement naturel d'ours."
 
- innovation_warbrave:0 "Braves-guerriers"
+ innovation_warbrave:0 "Braves-Guerriers"
```

```suggestion
 innovation_warbrave:0 "Braves-guerriers"
```
https://www.wowhead.com/fr/npc=97871/brave-guerrier-de-haut-roc
Il semblerait que le second mot soit une minuscule

---

Là je serais pour laisser une majuscule, en jeu ça fit mieux je trouve